src/test/java/nl/knaw/dans/ingest/core/service/mapper/MappingIntegrationTest.java

```diff
@@ -193,4 +199,32 @@ void DD_1216_provenance_maps_to_notes() throws Exception {
             .filter(f -> f.getTypeName().equals(NOTES_TEXT)).findFirst().orElseThrow();
         assertThat(field.getValue()).isEqualTo("copied xml to csv");
     }
+
+    @Test
+    void DD_1265_subject_omits_other() throws Exception {
```

Comment:
@jo-pol: I'll merge it, but I  think it would make things more readable if MinimalDdm were a helper class (nothing complex of course) that let's you do something like this:
```Java
var ddm = new MinimalDdm();
ddm.replaceAudience("D19200", "DD11200");  // etc.
var result = mapDdmToDataset(ddm.toString());
```